instructions-for-devs.md

```diff
@@ -78,7 +78,7 @@ Options:
 
 ## 3. Interact with `xud` via Command-Line Interface
 
-Interact with an `xud` process, identified by its `rpc` host and port using `xucli`. For getting up-to-date CLI commands, use `./xucli --help` as shown below:
+Open a new terminal (make sure `xud` is running either in a different terminal or as [service](#daemonize-xud)) and interact with `xud`, identified by its `rpc` host and port using `xucli`. For getting up-to-date CLI commands, use `./xucli --help` as shown below:
```

Comment:
The wording seems a a bit excessive to me. We say to open a new terminal in the previous section already. Having the daemonize instructions are good (although it should say "as **a** service") but I think they should go in **Start xud**. It's also possible to interact with xud on another machine, which this doesn't make clear. 

Maybe we can say "Interact with a *running* xud process" if you feel it needs more clarification, but I think it's OK as is.

Reply:
Well not clear enough obviously, as we could see some user had an issue understanding that `xud` needs to be running in a different terminal/process in order for `xucli` to interact with it. But agree on the excessive wording. Better?